debezium-core/src/test/java/io/debezium/junit/ConditionalFail.java

```diff
@@ -21,12 +24,26 @@
  */
 public class ConditionalFail extends AnnotationBasedTestRule {
 
+    private static final Logger FLAKY_LOGGER = LoggerFactory.getLogger(Flaky.class);
+
+    private static final String JIRA_BASE_URL = "https://issues.jboss.org/browse/";
```

Comment:
I know it's nit-pick but we probably should use the new domain `issues.redhat.com` here rather than the old redirect address, who knows how long that redirect will last for.

Reply:
Thanks, done. I copied it for very old history, not sure why :-)